lua/zk/init.lua

```diff
@@ -1,29 +1,15 @@
-local zk = {}
+_G.zk_util = require("zk.util")
```

Comment:
do we still need this global utility? not using the more complicated way to create commands previously used.